src/ui/battle-message-ui-handler.ts

```diff
@@ -1,13 +1,12 @@
 import BattleScene from "../battle-scene";
 import { addBBCodeTextObject, addTextObject, getTextColor, TextStyle } from "./text";
 import { Mode } from "./ui";
-import * as Utils from "../utils";
 import MessageUiHandler from "./message-ui-handler";
-import { getStatName, Stat } from "../data/pokemon-stat";
 import { addWindow } from "./ui-theme";
 import BBCodeText from "phaser3-rex-plugins/plugins/bbcodetext";
 import {Button} from "#enums/buttons";
 import i18next from "i18next";
+import { Stat, PERMANENT_STATS, getStatKey } from "#app/enums/stat.js";
```

Comment:
```suggestion
import { Stat, PERMANENT_STATS, getStatKey } from "#app/enums/stat";
```
NIT: Get rid of extension